web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.html

```diff
@@ -57,7 +57,7 @@
     <mat-form-field layout-fill appearance="outline">
       <input
         matInput
-        [(ngModel)]="this.step.options.output_permissions"
+        [(ngModel)]="this.step.options.outputPermissions"
```

Comment:
To avoid conflicts I suggest rollback ingest options. My PR will take care of that.